ReflectometryServer/test_modules/data_mother.py

```diff
@@ -1,6 +1,8 @@
 """
 SimpleObservable data and classes.
 """
+from __future__ import absolute_import, division
```

Comment:
Don't think this is required here. Python 3 will enforce absolute import anyways.